vpu/generic/vpuGeneric/.doc/README.md

```diff
@@ -0,0 +1,27 @@
+# Generic Dockerfile Template Specific Documentation
```

Comment:
Do we add this as part of the first commit?

Reply:
I think so, as it comes like that from the template. Then, any change you make on this document goes in the third commit.

Reply:
Noted, thanks!